Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -103,13 +103,52 @@ type Props = $ReadOnly<{|
   barStyle?: ?('default' | 'light-content' | 'dark-content'),
 |}>;
 
+type StackEntryProps = {
+  /**
+   * The background color of the status bar.
+   *
+   * @platform android
+   */
+  backgroundColor: {
+    value: ?string,
+    animated: ?boolean,
+  },
+  /**
+   * Sets the color of the status bar text.
+   */
+  barStyle: {
+    value: ?string,
+    animated: ?boolean,
+  },
+  /**
+   * If the status bar is translucent.
+   * When translucent is set to true, the app will draw under the status bar.
+   * This is useful when using a semi transparent status bar color.
+   */
+  translucent: ?boolean,
+  /**
+   *
+   */
+  hidden: {
+    value: ?boolean,
+    animated: boolean,
+    transition: ?('slide' | 'fade'),
+  },
+  /**
+   * If the network activity indicator should be visible.
+   *
+   * @platform ios
+   */
+  networkActivityIndicatorVisible: ?boolean,
+};
+
 /**
  * Merges the prop stack with the default values.
  */
 function mergePropsStack(
-  propsStack: Array<Object>,
-  defaultValues: Object,
-): Object {
+  propsStack: Array<StackEntryProps>,
```

Comment:
We should be able to use `$ReadOnlyArray<StackEntryProps>` here.

Reply:
Thanks review.
I changed `Array<StackEntryProps>` to  `$ReadOnlyArray<StackEntryProps>` and `npm run flow` at local, it actually ok.